### ISSUE TITLE
No villagers spawn in a village

### ISSUE THREAD

**Issue by loveyanbei** [cuberite/cuberite#3273]
Client version:   1.9.4
Server OS:        Linux
Commit id:        a2a9341c24609e45e563427eecc1a927ee2b657a

### Expected behavior
Found a village in gamemode 0,excepted some villagers

### Actual behavior
No villagers there

### Steps to reproduce the behavior
simply found a village in the game

**Comment by fralix** [comment]
I guess villagers are not yet in the ai, right? They are only to place / summon and trading interface doesn't work as well, as far as I know

**Comment by madmaxoft** [comment]
Exactly. The villagers aren't implemented properly yet.

**Comment by mhsjlw** [comment]
Yeah, trading interfaces don't currently work either, making them useless even if they did spawn.
